pkg/github/github.go

```
@@ -26,6 +29,42 @@ type Github struct {
 	Client *github.Client
 }
 
+// ListTeamMembers lists members of a github team
+func (g *Github) ListTeamMembers(team client.GithubTeam) ([]client.GithubTeamMember, error) {
+	ctx := g.Ctx
+	teamID := team.TeamID
+
+	org, _, err := g.Client.Organizations.Get(ctx, team.Organisation)
+	if err != nil {
+		return nil, err
+	}
+
+	users, _, err := g.Client.Teams.ListTeamMembersByID(ctx, *org.ID, teamID, nil)
+	if err != nil {
+		fmt.Println(err)
```

I know this is WIP, but just adding a note here, should return the err here